packages/client/src/components/notebook/AddVariablePopover.tsx

```diff
@@ -150,6 +152,41 @@ export const AddVariablePopover = observer((props: AddVariablePopoverProps) => {
 
     let alreadyAliased = false;
 
+    // TODO: Potentially Look into adding json viewer library for removing Editor.
```

Comment:
Do we want to keep commented these instances of commented out code?

Reply:
Removed commented out code, left TODO.